### ISSUE TITLE
Clarification for Lr Scheduler ReduceLROnPlateau in PL

### ISSUE THREAD

**Issue by JonnyD1117** [PyTorchLightning/pytorch-lightning#2280]
Maybe I missed it in PL documentation; however, when using the learning rate scheduler " ReduceLROnPlateau" appears to take an argument for which metric to monitor during training. 

Does PL handle passing this argument? If so is there anyway for the user to specify which argument is passed to the scheduler?

From Pytorch Docs, example shows scheduler.step takes val_loss ,as the argument for when to reduce learning rate for this particular scheduler.  

Example
>>> optimizer = torch.optim.SGD(model.parameters(), lr=0.1, momentum=0.9)
>>> scheduler = ReduceLROnPlateau(optimizer, 'min')
>>> for epoch in range(10):
>>>     train(...)
>>>     val_loss = validate(...)
>>>     # Note that step should be called after validate()
>>>     scheduler.step(val_loss)

**Comment by JonnyD1117** [comment]
After triple checking documentation, I found a NOTE that I missed initially, which address this issue. 

Sorry to bother.
